spec-tests/src/harness/config/index.ts

```diff
@@ -0,0 +1,46 @@
+import {getEnv} from "./getEnv";
+import assert from "node:assert";
+import {privateKeyToAccount} from "viem/accounts";
+import {Account} from "viem";
+
+export function loadHarnessConfig():HarnessConfig {
+  const env = getEnv()
+  const nodeType = env.string('HARNESS_NODE')
+  assert(nodeType === 'anvil' || nodeType === 'tenderly', 'HARNESS_NODE must be either anvil or tenderly')
+
+  const node: HarnessNode =
+    nodeType === 'anvil'
+      ? {
+        mode: 'anvil',
+        alchemyApiKey: env.string('ALCHEMY_API_KEY'),
+      }
+      : {
+        mode: 'tenderly',
+        tenderlyApiKey: env.string('TENDERLY_API_KEY'),
+        tenderlyAccount: env.string('TENDERLY_ACCOUNT'),
+        tenderlyProject: env.string('TENDERLY_PROJECT'),
```

Comment:
Hi @krzkaczor! I was wondering what you think about having only the virtual testnet admin URL as part of the configuration. Similar to `AnvilNode` with `ALCHEMY_API_KEY`, we could use something like `TENDERLY_ADMIN_RPC_URL`. 
Do you think that creation process of the virtual testnet might be a bit too much for the test suite?

Reply:
Right now test harness spawns a new node instance for each test. It's slow but it's bullet proof as we don't need to rely on snapshots or anything. To spawn anvil we need need some kind of RPC url, to spawn vnet we need API access, hence tenderly credentials are required.